README.adoc

```diff
@@ -141,23 +145,21 @@ openlibertydumps          oldump,oldumps     openliberty.io   true         OpenL
 openlibertytraces         oltrace,oltraces   openliberty.io   true         OpenLibertyTrace
 ```
 
-The Open Liberty Operator introduces new
-https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[CRDs^],
-including the `OpenLibertyApplication`. 
+The Open Liberty Operator introduces new CRDs,including the `OpenLibertyApplication` CRD. 
 The `OpenLibertyApplication` CRD defines a set of configurations for
 deploying an Open Liberty-based application, including the application image, number of instances, and storage settings.
-The Open Liberty Operator watches changes to instances of the OpenLibertyApplication and creates Kubernetes resources based on the config defined in the CRD instance.
+The Open Liberty Operator watches for changes to instances of the OpenLibertyApplication CRD and creates Kubernetes resources that are based on the configuration that is defined in the CRD instance.
 
-You will be using the `OpenLibertyApplication` resource in the upcoming sections.
+You can use the `OpenLibertyApplication` resource in the upcoming sections.
```

Comment:
I think it is more accurate to leave it as "you will be using the `OpenLibertyApplication` resource"